models/subreddit.es6.js

```diff
@@ -38,10 +38,6 @@ class Subreddit extends Base {
   static fields = EDIT_FIELDS;
 
   constructor(props) {
-    delete props.submit_text_html;
-    delete props.description_html;
-    delete props.public_description_html;
-
```

Comment:
you should delete the markdown properties instead, now that we're using the html - that will reduce the size of the json payload written to the page.